cmds/webboot/webboot.go

```diff
@@ -16,10 +20,12 @@ var (
 	v       = flag.Bool("verbose", false, "Verbose output")
 	verbose = func(string, ...interface{}) {}
 	dir     = flag.String("dir", "", "Path of cached directory")
-	network = flag.Bool("network", true, "Should or not set up network")
-	boot    = flag.Bool("boot", true, "Should or not boot the iso. May trun it off for test.")
+	network = flag.Bool("network", true, "If dry-network is false we will not set up network")
+	dryRun  = flag.Bool("dry-run", true, "If dry-run is true we won't boot the iso. May trun it on for test.")
```

Comment:
what does trun mean? test run? or run?

Reply:
I think my expression is wrong. what I want to mean is, we may use this args in test, when we don't want to actually replace the kernel. I may  just say "If dry-run is true we won't boot the iso is ok."